sdk/resources/arm-resources/package.json

```diff
@@ -17,17 +24,23 @@
     "isomorphic"
   ],
   "license": "MIT",
-  "main": "./dist/arm-resources.js",
-  "module": "./esm/resourceManagementClient.js",
-  "types": "./esm/resourceManagementClient.d.ts",
+  "main": "./dist/index.js",
+  "module": "./esm/index.js",
+  "types": "./esm/index.d.ts",
```

Comment:
```suggestion
  "types": "./types/arm-resources.d.ts",
```

Reply:
@deyaaeldeen Same here

Reply:
No correctness issues here, my suggestion is purely to be consistent with data-plane packages.